### PR TITLE
Serialize logic

### PR DESCRIPTION
**Pull request by shane-tomlinson**
@TDA - can you review this?

### CODE REVIEW

File: src/passwordcheck.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ define(['bloomfilter',
         PASSWORD_TOO_SHORT: 'PASSWORD_TOO_SHORT'
       };
 
-      function strengthCheck(password) {
+      function isStrongEnough(password) {
         // Check for passwords that at least contain a number and an alphabet,
         // or if alphabets, then at least (minLength + 4) characters long
         var regexString = '((?=.*[a-zA-Z])(?=.*[0-9])[A-Za-z0-9!@#$%^&*()_+ ]{' + minLength + ',' + (minLength + 4) + '})|([A-Za-z0-9!@#$%^&*()_+ ]{' + (minLength + 4) + ',})';
```
Comment (shane-tomlinson) [inline src/passwordcheck.js:34]: @TDA - I think we should change this regexp to check if the string is _only_ characters and numbers. If so, check that it is fewer characters than minLength + 4. That'll significantly simplify the regexp, and it'll allow international users to use their special characters as part of the password that do not fit into this list.

**Comment by TDA** [comment]
Looks great @shane-tomlinson. Merging!

File: src/passwordcheck.js (hunk 1 above)
Comment (TDA) [inline src/passwordcheck.js:34]: @shane-tomlinson I believe that would be a good change, but we should also change the name of the function to better reflect that, maybe 'isWeakPassword'? 'Cause, right now we are whitelisting, the change would be checking for weak ones only, is that right?

Comment (shane-tomlinson) [inline src/passwordcheck.js:34]: @TDA - Yup, you are right.

**Pull request merged**